iOS/Scenes/Manga/MangaViewController.swift

```diff
@@ -540,7 +540,29 @@ extension MangaViewController {
 
     private func makeMenu() async -> [UIMenuElement] {
         var menus = [UIMenu]()
-        var actions = [
+        var actions: [UIMenuElement] = [
+            // mark as read & mark as unread submenu
+            UIMenu(title: NSLocalizedString("MARK_ALL", comment: ""), image: UIImage(systemName: "pencil.tip.crop.circle"),children: [
+                // read chapters
+                UIAction(title: NSLocalizedString("READ", comment: "")) { _ in
+                    self.showLoadingIndicator()
+                    let chapters = [Chapter](self.viewModel.chapterList)
```

Comment:
why do you need to cast this? just store it

Reply:
removed casting, directly used in the function